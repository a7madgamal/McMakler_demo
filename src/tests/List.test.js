import React from 'react';
import renderer from 'react-test-renderer';
import List from '../components/List'
it('renders correctly: no props', () => {
  const tree = renderer.create(<List items={[]}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
