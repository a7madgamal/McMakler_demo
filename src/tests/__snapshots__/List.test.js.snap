// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly: no props 1`] = `
<div
  className="row"
>
  <ul
    className="list "
  />
</div>
`;
